src/i18n/cs/plugins/exec/sim-host-dialogs.html

```diff
@@ -4,7 +4,7 @@
 <cordova-dialog id="exec-dialog" caption="Neošetřené volání metody Exec" data-loc-id="f7283850">
     <cordova-group>
         <p data-loc-id="a1a63f26">Pro následující volání metody Exec není k dispozici obslužná rutina:</p>
-        <p style="font-style:italic"><span id="exec-service" data-loc-id="ba2b4c80">služba</span>.<span id="exec-action" data-loc-id="e4c1ba28">akce</span></p>
+        <p style="font-style:italic"><span id="exec-service" data-loc-id="ba2b4c80">služba</span>.<span id="exec-action" data-loc-id="e4c1ba28">akce</span>(<span id="exec-args" data-loc-id="veyymo5t">arguments</span>)</p>
```

Comment:
These are generated by a tool. It looks like @dmitrii-snytkin ran the tools, however (based on the changes that were generated).

However, usually (to make code reviews easier, apart from anything else), I don't run the tool until I'm about to do a release (basically: let a bunch of changes accumulate, then when about to do a release, run the tool, then notify the translation team, they do their changes in one go, then do the release).

So, @dmitrii-snytkin, maybe remove the `i18n` changes from the PR.